src/App.jsx

```diff
@@ -1,12 +1,26 @@
-import { JobContext } from './context';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import HomePage from './components/HomePage';
+import RecentJobs from './components/RecentJobs';
+import SearchArea from './components/SearchArea';
+
+import { JobContextProvider } from './context';
+import NavMenu from './components/NavMenu';
 
 function App() {
   return (
-    <>
-      <JobContext.Provider>
-        <div>Main</div>
-      </JobContext.Provider>
-    </>
+    <div className="font-sans font-normal text-base bg-slate-50">
+      <JobContextProvider>
+        <NavMenu />
```

Comment:
You should move the NavMenu into the BrowserRouter - you won't be able to use Link or NavLink outside

Reply:
done